modifications en amont de l'ago du 30 mars

iv-stpn opened the pull request:
Signatires : @nada912 @melisandeGB 
Reviewers : @gtedavid @noftaly

iv-stpn committed:
modifications en amont de l'ago du 30 mars

iv-stpn committed:
nom du règlement intérieur corrigé, README mis à jour

melisandeGB committed:
Signature Mélisande

Commit:
Signature Nada

noftaly reviewed (changes requested):
Définir "Quorum"

noftaly commented on Règlement Intérieur d'Horizon EFREI.md:
```diff
@@ -0,0 +1,266 @@
+
+Les Documents Officiels de l'Association Horizon EFREI incluent :
+
+- <u>Les Statuts</u>, un document unique, susceptible d'être modifié au cours de la vie de l'Association, qui vient compléter les articles de la loi du 1er juillet 1901 pour définir légalement l'Association et statuer sur la marche à suivre pour la bonne réalisation de ses activité ;
```
```suggestion
- <u>Les Statuts</u>, un document unique, susceptible d'être modifié au cours de la vie de l'Association, qui vient compléter les articles de la loi du 1er juillet 1901 pour définir légalement l'Association et statuer sur la marche à suivre pour la bonne réalisation de ses activités ;
```

iv-stpn committed:
correction orthographe

Co-authored-by: Elliot <elliot.maisl@gmail.com>

iv-stpn committed:
mises à jour suite à l'AGO du 30 mars

- ajout de la mention "les Statuts doivent être lus et acceptés"
- déplacement de la section sur les Clubs Associatifs dans les Statuts
- ajout des Lettres Officielles
- ajout du Procès-Verbal d'AGO
- ajout des signatures et clarification quant aux signatures dans le RI
- ajouter de la mention des cinq Membres Adhérents pour convier une AGO
- corrections orthographiques et errata

iv-stpn commented:
> Définir "Quorum"

Le mot "quorum" n'est plus mentionné dans les Documents Officiels de l'Association
